File: flut/.flutter/lib/flutter/material/slider_theme.dart
import 'package:flutter/material.dart';
import 'package:flut/flut/object.dart';

class FlutShowValueIndicator extends FlutEnumObject<ShowValueIndicator> {
  const FlutShowValueIndicator()
    : super('ShowValueIndicator', const {
        'onlyForDiscrete': ShowValueIndicator.onlyForDiscrete,
        'onlyForContinuous': ShowValueIndicator.onlyForContinuous,
        'always': ShowValueIndicator.always,
        'onDrag': ShowValueIndicator.onDrag,
        'alwaysVisible': ShowValueIndicator.alwaysVisible,
        'never': ShowValueIndicator.never,
      });
}

File: flut/flutter/material/slider_theme.py
from flut._flut.engine import FlutEnumObject


class ShowValueIndicator(FlutEnumObject):
    onlyForDiscrete: "ShowValueIndicator"
    onlyForContinuous: "ShowValueIndicator"
    always: "ShowValueIndicator"
    onDrag: "ShowValueIndicator"
    alwaysVisible: "ShowValueIndicator"
    never: "ShowValueIndicator"
